assets/constants.js

```diff
@@ -18,5 +18,214 @@ var constants = {
     "Statistics", "Trigonometry", "Welcome"],
   "DISABLED_EXPLORATION_IDS": ["5"],
```

Comment:
Could you move the following to the top of the file:

- CAN_SEND_ANALYTICS_EVENTS
- ENABLE_PROMO_BAR
- SITE_NAME
- SITE_FEEDBACK_FORM_URL

since these tend to get modified during release? Then have the other constants below.

/cc @wxyxinyu